mondaynet/values.yaml

```diff
@@ -126,25 +126,33 @@ activation:
     cache_script_size: 100000000
     cache_stake_distribution_cycles: 8
     cache_sampler_state_cycles: 8
-    tx_rollup_enable: true
-    tx_rollup_origination_size: 4000
-    tx_rollup_hard_size_limit_per_inbox: 500000
-    tx_rollup_hard_size_limit_per_message: 5000
-    tx_rollup_max_withdrawals_per_batch: 15
-    tx_rollup_commitment_bond: "10000000000"
-    tx_rollup_finality_period: 10
-    tx_rollup_max_inboxes_count: 15
-    tx_rollup_withdraw_period: 10
-    tx_rollup_max_messages_per_inbox: 1010
-    tx_rollup_max_commitments_count: 30
-    tx_rollup_cost_per_byte_ema_factor: 120
-    tx_rollup_max_ticket_payload_size: 2048
-    tx_rollup_rejection_max_proof_size: 30000
-    tx_rollup_sunset_level: 17280
-    sc_rollup_enable: true
-    sc_rollup_origination_size: 6314
-    sc_rollup_challenge_window_in_blocks: 40
-    sc_rollup_max_available_messages: 1000000
+    tx_rollup:
+      enable: true
+      origination_size: 4000
+      hard_size_limit_per_inbox: 500000
+      hard_size_limit_per_message: 5000
+      max_withdrawals_per_batch: 15
+      commitment_bond: "10000000000"
+      finality_period: 10
+      max_inboxes_count: 15
+      withdraw_period: 10
+      max_messages_per_inbox: 1010
+      max_commitments_count: 30
+      cost_per_byte_ema_factor: 120
+      max_ticket_payload_size: 2048
+      rejection_max_proof_size: 30000
+      sunset_level: 17280
+    sc_rollup:
+      enable: true
+      origination_size: 6314
+      challenge_window_in_blocks: 40
+      max_available_messages: 1000000
+      stake_amount: "32000000"
+      commitment_period_in_blocks: 20
+      max_lookahead_in_blocks: 30000
+      max_active_outbox_levels: 20160
+      max_outbox_messages_per_level: 100
+      number_of_sections_in_dissection: 32
```

Comment:
mondaynet starts from Jakarta and updates to alpha so it's probably not needed yet?